install.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# CHECKS FOR SUDO ACCESS (added because of those who have seperate partitions for /home and /root) 
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please don't run as root" && exit
+  exit
+fi
+
+# MAKE DIRECTORY SO THAT YOUR HOME DIRECTORY DOESN'T GET DIRTY
+
+cd /home/$USER
+mkdir -p junk-drawer
+cd junk-drawer
+
+# Install git and wget
+
+echo "What distro do you use?"
+echo "1) Arch Linux"
+echo "2) Fedora Linux"
+echo "3) Gentoo/Funtoo Linux"
+echo "4) Ubuntu/Debian and Ubuntu/Debian based distros"
+echo "5) Void Linux"
+echo "6) NixOS"
+echo "7) Alpine Linux"
+echo "8) openSUSE Linux"
+read -p "Please input a number between 1 and 8: " DISTRO
+case $DISTRO in 
+[1])
+	sudo pacman -S git wget --needed
+	;;
+
+[2])
+  sudo dnf install wget git
+	;;
+
+[3])
+  sudo emerge wget git
+  ;;
+
+[4])
+	sudo apt install wget git
+	;;
+  
+[5])
+  sudo xbps-install wget git
+  ;;
+  
+[6])
+  sudo nix-shell -p wget git
+  ;;
+  
+[7])
+  sudo apk add wget git
+  ;;
+  
+[8])
+  sudo zypper install wget git
+  ;;
+esac
+
+# Download, unzip, and execute SPF (formerly Steam Friends Patcher)
+
+wget https://github.com/PhantomGamers/SFP/releases/download/0.0.14/SFP_UI-SelfContained-linux-x64.zip
+unzip SFP_UI-SelfContained-linux-x64.zip
```

Comment:
`unzip` is not commonly available, please also add it to the install checks. This step would fail on my NixOS.

Reply:
IT'S NOT???

Reply:
> `unzip` is not commonly available, please also add it to the install checks. This step would fail on my NixOS.

Fixed it